dask_cuda/utils.py

```diff
@@ -129,45 +129,61 @@ def get_device_total_memory(index=0):
     ).total
 
 
-def get_ucx_env(enable_infiniband=False, interface=None, enable_nvlink=False):
+def get_ucx_env(
+    enable_tcp=True, enable_infiniband=False, interface=None, enable_nvlink=False
+):
```

Comment:
I like that this is pulled off into a separate function.  This seems like the sort of thing that could be moved upstream easily.

(athough hopefully we can do this programmatically at some point)

Reply:
Yup, hopefully most of the work here (and the upcoming PR) will be easily moveable.